Submissions/Bain_Week 3.playground/Contents.swift

```diff
@@ -0,0 +1,227 @@
+//: ## Functions and Closures
+//: 1.)
+//: Remove the `day` parameter. Add a parameter to include today’s lunch special in the greeting.
+//:
+//:  Write a custom argument label before the parameter name, or write `_` to use no argument label.
+//:
+func greet(_ person: String, lunchSpecial: String) -> String {
+    return "Hello \(person), today's lunch special is \(lunchSpecial)."
+}
+greet("Bob", lunchSpecial: "tacos")
+
+//: 2.)
+//:
+//: Rewrite the closure to return zero for all odd numbers.
+//:
+var numbers = [20, 19, 7, 12]
+
+//: full "syntactic sugar" form:
+numbers.map({ (number: Int) -> Int in
+    let result = number%2 == 0 ? 3 * number : 0
+    return result
+})
+//: omitting parameters and return type form:
+let mappedNumbers = numbers.map({ number in 3 * number })
+print(mappedNumbers)
+//: reference variable form:
+let mappedNums = numbers.map { 3 * $0 }
+print(mappedNums)
+//: ## Objects and Classes
+//: 3.)
+//:
+//: Add a constant property with `let`, and add another method that takes an argument.
+//:
+//: 4.)
+//:
+//: This version of the `Shape` class is missing something important: an initializer to set up the class when an instance is created. Use `init` to create one.
```

Comment:
The question requested that you use `init` to initialize the class instead of providing a default value for `size`.